src/pages/NPSP_Resources.page

```diff
@@ -3,7 +3,7 @@
     <head>
         <meta charset="utf-8"/>
         <meta http-equiv="X-UA-Compatible" content="IE=edge"/>
-        <title>Getting Started with the NPSP</title>
+        <title>Get Started with the NPSP</title>
```

Comment:
This is the current (old) Getting Started page, and should remain as "Getting Started".